persistence/remote/src/main/java/org/infinispan/persistence/remote/RemoteStore.java

```diff
@@ -514,16 +514,9 @@ private ConfigurationBuilder buildRemoteConfiguration(RemoteStoreConfiguration c
          ;
       }
 
-      Properties propertiesToUse;
-      Properties actualProperties = configuration.properties();
-      if (!actualProperties.contains("blocking")) {
-         // Need to make a copy to not change the actual configuration properties
-         propertiesToUse = new Properties(actualProperties);
```

Comment:
initializer with default values, requires consumers to fetch values not check if they are present